### PR TITLE
Resume vertical slider in the gallery

### PR DESCRIPTION
**Pull request by arnaudmiribel**
Had to blacklist vertical slider because of a bug in the gallery, but that was fixed by the author. So, kinda reverting!

[Demo gallery app](https://app-extras-xvbagqztvfhbybcuaewtzf.streamlit.app/)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -24,7 +24,7 @@ streamlit-embedcode = ">=0.1.2"
 streamlit-keyup = ">=0.1.9"
 protobuf = "!=3.20.2"
 streamlit-camera-input-live = ">=0.2.0"
-streamlit-vertical-slider = ">=1.0.2"
+streamlit-vertical-slider = ">=2.5.5"
```
Comment (arnaudmiribel) [inline pyproject.toml:27]: https://pypi.org/project/streamlit-vertical-slider/#history

**Comment by blackary** [comment]
Need to update the lockfile

**Comment by sfc-gh-amiribel** [comment]
Thanks, done @blackary.

**Review by blackary — APPROVED** [review]
Ship it!

**Pull request merged**